Change in Wazuh documentation Ansible (#4479)
wazuh/wazuh-documentation: source/deploying-with-ansible/roles/wazuh-opendistro.rst

.. Copyright (C) 2020 Wazuh, Inc.

.. _ansible-wazuh-opendistro:

> Open Dstro

Reviewer: ```suggestion
Open Distro
```

Author: This suggestion is out of date. The requested change is already present:
  - https://github.com/wazuh/wazuh-documentation/pull/4479/files

-----------

> This role is intended to deploy Open Distro node, you have some variables that can be used to customize the installation:

Reviewer 2: ```suggestion
This role is intended to deploy the Open Distro node. The following variables can be used to customize the installation:
```

> This role is intended to deploy Opendistro node, you have some variables that can be used to customize the installation:

Author: This suggestion is out of date. The requested change is already present:
  - https://github.com/wazuh/wazuh-documentation/pull/4479/files

  $ ansible-playbook wazuh-elastic.yml -e@vars-production.yml

> The example above will install Opendistro and set the listen address to: ``10.1.1.10`` using ``vars-production.yml``.

Reviewer: ```suggestion
The example above will install Open Distro and set the listen address to: ``10.1.1.10`` using ``vars-production.yml``.
```

Author: This suggestion is out of date. The requested change is already present:
  - https://github.com/wazuh/wazuh-documentation/pull/4479/files
